.gitignore

```diff
@@ -7,8 +7,9 @@
 .pt2
 
 # directories
+__pycache__
```

Comment:
This is taken care of by `*.py[cod]` above...

Reply:
In deed, in that case all my changes were included in a commit yesterday, we can close this PR